examples/inference-deployments/mpt/README.md

```diff
@@ -1,34 +1,130 @@
+    - `mpt_7b.yaml` - a yaml to deploy [MPT-7B Base](https://huggingface.co/mosaicml/mpt-7b).
+    - `mpt_7b_instruct.yaml` - a yaml to deploy [MPT-7B Intstruct](https://huggingface.co/mosaicml/mpt-7b-instruct).
+    - `mpt_7b_storywriter.yaml` - a yaml to deploy [MPT-7B StoryWriter](https://huggingface.co/mosaicml/mpt-7b-storywriter).
+- Model handlers - these define how your model should be loaded and how the model should be run when receiving a request. You can use the default handlers here or write your custom model handler as per instructions [here](https://docs.mosaicml.com/projects/mcli/en/latest/inference/deployment_features.html#custom-model-handlers).
+    - `mpt_7b_handler.py` - a model handler using DeepSpeed.
+    - `mpt_7b_ft_handler.py` - a model handler using FasterTransformer.
+- `requirements.txt` - package requirements to be able to run these models.
+
+
+## Setup
+
+Please follow instructions in the Inference Deployments [README](https://github.com/mosaicml/examples/tree/main/examples/inference-deployments/README.md) and make sure 
+- You have access to our inference service.
+- Your dev environment is set up with `mcli`.
+- You have a cluster to work with.
+
+## Deploying your model
 
 To deploy, simply run `mcli deploy -f mpt_7b_instruct.yaml --cluster <your_cluster>`.
 
-## Input
+Once deployed, you can ping the deployment using
+```python
+from mcli import ping
+ping('deployment-name')
+```
+to check if it is ready (status 200).
+
+More instructions can be found [here](https://docs.mosaicml.com/projects/mcli/en/latest/quick_start/quick_start_inference.html)
 
+### Deploying from cloud storage
+If your model exists on Amazon S3 or Hugging Face, you can edit the YAML's model params to deploy it:
+```yaml
+model:
+    download_parameters:
+        s3_path: <your-s3-path>
+    model_parameters:
+        ...
+        model_name_or_path: my/local/s3_path
 ```
-{
+
+If your model exists on a different cloud storage, then you can follow instructions [here](https://docs.mosaicml.com/projects/mcli/en/latest/inference/deployment_features.html#id1) on writing your custom downloader function, and deploy the model.
+
+## Sending requests to your deployment
+
+Once the deployment is ready, it's time to run inference! 
+
+<details>
+<summary> Using Python SDK </summary>
+
+
+```python
+from mcli import predict
+
+deployment = get_inference_deployment(<deployment-name>)
+input = {
     {
         "input_strings": "Write 3 reasons why you should train an AI model on domain specific data set.",
         "temperature": 0.01
     }
 }
+predict(deployment, input)
+
+```
+</details>
+
+<details>
+<summary> Using MCLI </summary>
+
+```bash
+mcli predict <deployment-name> --input '{"input_strings": ["hello world!"]}'
+
+```
+</details>
+
+<details>
+<summary> Using Curl </summary>
+
+```bash
+curl https://<deployment-name>.inf.hosted-on.mosaicml.hosting/predict_stream \
```

Comment:
should this example be with predict stream? just not sure if that was intentional or not